integrations/livepeer/components/stream.tsx

```diff
@@ -0,0 +1,49 @@
+'use client'
+
+import { useMemo, useState } from 'react'
+
+import { Player, useCreateStream } from '@livepeer/react'
+
+import { Button } from '@/components/ui/button'
+import { Input } from '@/components/ui/input'
+
+export const Stream = () => {
+  const [streamName, setStreamName] = useState<string>('')
+  const { mutate: createStream, data: stream, status } = useCreateStream(streamName ? { name: streamName } : null)
+
+  const isLoading = useMemo(() => status === 'loading', [status])
+  return (
+    <div>
+      <label>
+        Enter a stream name
+        <Input style={{ margin: '10px 0px' }} type="text" placeholder="Stream name" onChange={(e) => setStreamName(e.target.value)} />
```

Comment:
The project is using tailwind for stylling, so it would be ideal to replace inline styles by tailwind utility classes, more info on tailwind [here.](https://tailwindcss.com/)